test/kubeclient_test.rb

```diff
@@ -1,7 +1,4 @@
-require 'minitest/autorun'
-require 'json'
-require 'webmock/minitest'
-require './lib/kubeclient'
+require './test/test_helper'
```

Comment:
See http://ruby-doc.org//stdlib-2.0/libdoc/rake/rdoc/Rake/TestTask.html .  You may need to set the testfiles...I thought that was done automatically, but maybe not?